src/stores/UserPreferencesStore.js

```diff
@@ -17,6 +17,7 @@ let _defaults = {
     SpeechRate: 1,
     SpeechPitch: 1,
     TTSLanguage: 'en-US',
+    UserName: '',
```

Comment:
Please don't mind the codacy error. Its occurring due to using single quotes for strings which is preferable and also required for travis. Thanks.
![e1](https://user-images.githubusercontent.com/30981465/41364030-03e760f8-6f53-11e8-86c8-b511dba96e28.png)